test/extended/dr/quorum_restore.go

```diff
@@ -134,16 +147,14 @@ var _ = g.Describe("[sig-etcd][Feature:DisasterRecovery][Disruptive]", func() {
 							failures = 0
 						}
 
-						// there is a small chance the cluster restores the default replica size during
```

Comment:
I think it's desirable to keep this check here rather than externalizing in a goroutine. It's already in a polling loop, I was just asking to have any error it returned logged for traceability.